backend/model/south-connector.model.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * DTO for South connectors
+ */
+export interface SouthConnectorDTO {
+  id: string;
+  name: string;
+  type: string;
+  description: string;
+  enabled: boolean;
+  settings: object;
```

Comment:
this doesn't look too good to me, we should have types for settings with inheritance

Reply:
I don't think it will be compatible with a plugin approach ?

Reply:
I suggest to keep it like that for now, and manage that in a future PR when I'll change the schema on the backend

Reply:
we should make it compatible i don't see the reason it shouldn't be the plugins should import our types and extend accordingly the base class

Reply:
if we don't have a model for settings then we have no type safety for south connectors which doesn't seem to good for me. ok to take that into account in a subsequent pr.